kahuna/public/js/search/query.js

```diff
@@ -95,6 +98,7 @@ query.controller('SearchQueryCtrl',
 
     function resetQueryAndFocus() {
         ctrl.filter.query = '';
+        ctrl.filter.orderBy = undefined;
```

Comment:
This and the parameter in the label link seems rather brittle. There are many more links to search.results that would preserve the orderBy and possibly cause confusion. Could we find a way to programmatically clear that flag whenever we navigate to a search state that has no collection in the query?